packages/frontend/src/home/contributors.json

```diff
@@ -44,7 +44,7 @@
     "location": "🌍"
   },
   {
-    "name": "Xiaohanxu1996",
+    "name": "xiaohanxu-nick",
```

Comment:
this change seems unrelated?

Reply:
@cheapsteak My pleasure. Yes, that diff is unrelated. While adding myself to the contributors list, I noticed `Xiaohanxu1996` changed username on GitHub to `xiaohanxu-nick`. Seemed appropriate to update that but maybe it should have been a separate PR?

Reply:
Confirm here: [https://github.com/cheapsteak/npmcharts.com/graphs/contributors](https://github.com/cheapsteak/npmcharts.com/graphs/contributors)

Reply:
ah, thank you 🙏